run_dir/design/projects.html

```diff
@@ -12,164 +12,175 @@
-            <div class="col-md-6 panel-group">
-              <div class="panel panel-default">
-                <div class="panel-heading">
-                  <h4 class="panel-title">
+      <div id="collapseOne" class="collapse" role="tabpanel" aria-labelledby="headingOne">
+        <div class="card-body">
+          <div class="row">
+          <div class="card-group">
+
+              <div class="card m-2" style="max-height: 45vh; min-height: 45vh">
+                <div class="card-header">
+                  <h4 class="card-title">
                     Filters
                   </h4>
                 </div>
-                <div class="panel-body">
+                <div class="card-body overflow-scroll">
                   <div class="form-group">
                     <div class="text-center">
-                    <label for="statusbtnBar1" class="control-label">Choose Status <button type="button" class="btn btn-xs btn-default" id="filterInfo">
-                      <span class="glyphicon glyphicon-info-sign"></span></button></label>
-                      <div id='displayInfo' style="display:none;" class="alert alert-info" role="alert">
-                        <span type="button" class="close" aria-label="Close"><span aria-hidden="true">&times;</span></span>
+                    <label for="statusbtnBar1" class="fw-bold pb-2">Choose Status <button type="button" class="btn btn-xs btn-outline-secondary" id="filterInfo">
+                      <span class="fa fa-info-circle"></span></button></label>
+                      <div id='displayInfo' style="display:none;" class="alert alert-info mt-1" role="alert">
                         <div class="text-left">
+                          <span type="button" class="btn-close float-right" aria-label="Close"></span>
                             The Project status/Dates relationships are as follows:
                             <ul>
-                              <li><b>Pending</b>: Do not have Open, Queue or Close Dates.</li>
-                              <li><b>Reception Control</b>: Have Open Date, but not Queue or Close Dates.</li>
+                              <li><b>Pending</b>: Does not have Open, Queue or Close Dates.</li>
+                              <li><b>Reception Control</b>: Has Open Date, but not Queue or Close Dates.</li>
                               <li><b>Needs Review</b>: May have Open and Queue Dates, but not Close Date.</li>
-                              <li><b>Ongoing</b>: Have Open and Queue Date, but not Close Date.</li>
-                              <li><b>Open</b>: Have Open Date.</li>
-                              <li><b>Closed</b>: Have Close Date. May have Open and Queue dates.</li>
-                              <li><b>Aborted</b>: Have Aborted date.</li>
+                              <li><b>Ongoing</b>: Has Open and Queue Dates, but not Close Date.</li>
+                              <li><b>Open</b>: Has Open Date.</li>
```

Comment:
Should it say that it does not have close or aborted date as well? If that is correct

Reply:
This is mostly for the date filtering below, where we don't have a field for filtering the Aborted date.